packages/search-in-workspace/src/common/search-in-workspace-interface.ts

```diff
@@ -143,3 +144,11 @@ export interface SearchInWorkspaceServer extends JsonRpcServer<SearchInWorkspace
 
     dispose(): void;
 }
+
+export const SearchInWorkspaceServiceExtension = Symbol('SearchInWorkspaceServiceExtension');
+
+export interface ISearchInWorkspaceService {
```

Comment:
this file to define a JSON-RPC protocol between client and server, it seems to be an extension to frontend service, not a part of protocol.